### PR TITLE
[READY]Adds a special repair module, and a snare projector to the peacekeeper cyborg.

### PR DESCRIPTION
**Pull request by silicons**
I was initially going to add a large amount of modules and upgrades and rework some things, but atomized PRs and I'm not done with them and I'm not going to be done with them for a while, so I'm going to try to make them not so shit to play as right now
:cl:
rscadd:Peacekeeper Cyborgs now have a repair droid module, and snare launcher. Snares fired will fall off their targets after 1.5 seconds, and the launcher has a delay of 2 seconds.
experimental: Cyborg self-repair modules updated, but still works the same.
\:cl:
Here, let me rephrase this
This won't make them useful
This will make them NOT AS USELESS

### CODE REVIEW

**Comment by Repukan** [comment]
>Snare someone people are chasing
>People come up and beat the shit out of them

Congrats, you just broke law 1

**Comment by kevinz000** [comment]
fixed

**Comment by GunHog** [comment]
You do not need to include the repair droid, they are a cheap and low tech requirement upgrade from the exofab.

**Comment by AdamElTablawy** [comment]
>flash someone people are chasing

>they catch up and kill him while he's stunned

congrats you just broke law 1

you could apply that to literally any module and anything they do so tbqh that's a dumb argument repu

**Comment by xxalpha** [comment]
More power Creep.

Stop beating around the Bush and just readd the secborg.

**Comment by GunHog** [comment]
The headmins would never allow it :(

**Comment by PKPenguin321** [comment]
>Stop beating around the Bush and just readd the secborg

All this tells me is that we should remove PK Borg if people are going to use it to justify seborgs returning

**Comment by kevinz000** [comment]
There's going to be less justification if it's not useless now is there?

**Comment by ExcessiveUseOfCobblestone** [comment]
Secborg 2.0 Beta testing

**Comment by Aloraydrel** [comment]
Just add them to the service module where they belong :^)

**Comment by Anonmare** [comment]
Just give the secborgs a swarmer style disabler and remove the stunbaton and replace it with a telebaton. Seriously, the disabler is literally the only reason people people dislike it and telebaton can be defeated with cigs.

**Comment by Shadowlight213** [comment]
@Anonmare 
The true issue with secborg really isn't the stun. It's their ability to cuff then drag to the brig.
Or, if they don't have cuffs, stun lock and drag to brig. If beepsky was coded to drag arrested perps to the brig, he would fill most of secborg duties.
Actually. Thinking about it, most of the major issues with secborg can be fixed if dragging is removed from them. Maybe that's the solution.

**Comment by Anonmare** [comment]
@Shadowlight213 I'd agree but the standard module has zipties and it hasn't received any complaints to my knowledge. The issue is the stunning and restrains. If you weaken the ability to stun, then the cuffs are less dangerous.

**Comment by kevinz000** [comment]
@Shadowlight213  at that point a biological officer is way better.
they don't have laws :^)

**Comment by kevinz000** [comment]
I love how making a PR saying I'm going to give pkborgs 5 new modules including a deflector shield and the ones in this one, as well as improved barrier fields/whatnot gets more upvotes and less downvotes then a PR with a quarter of the powercreep that PR had. :/

**Comment by kevinz000** [comment]
@GunHog the repair droid is "SPECIALLY BALANCED" for pkborgs
higher repair rate but very high power draw.

**Comment by Shadowlight213** [comment]
@kevinz000 the snare launcher is mainly the issue.
Shield was fine, because it pushes PK borg in the direction of wallborg (borg sits between bad guy and crew and takes all the attacks), snare launcher pushes them in the direction of sec borg.

**Comment by kevinz000** [comment]
@Shadowlight213 Maybe people didn't notice but the snare launcher was in that PR too, and it had normal projectile speed too. If that's the issue I'l take it out since it appears a lot of people have issues with it.

**Comment by ExcessiveUseOfCobblestone** [comment]
PK borg needs a device that surrounds an individual with retextured anti-breach shields [see: Narsie rune summoning] instead of "lol slow down a guy harming so the other one harming can kill him with ease".

**Comment by XDTM** [comment]
You guys do realize that slowing people is already part of his kit with the tiring solution? This is a shorter-lasting, ranged slow, and it _can_ be used to prevent harm in a chase situation. A shielding module would be nice, though, since PK borgs already have several other ways of impairing movement.

**Comment by Supermichael777** [comment]
The real problem with sec borgs is people never used and admins never enfoced Law 2 let me go

**Comment by KorPhaeron** [comment]
Where did this "robots can never impact my round in any way" meme start?

**Comment by ExcessiveUseOfCobblestone** [comment]
replace AI with drone-master is the only solution.

**Comment by optimumtact** [comment]
These robots were never intended to be useful, they were added to trigger secborg players, we should have just left them out.

**Comment by xxalpha** [comment]
>These robots were never intended to be useful, they were added to trigger secborg players, we should have just left them out.

Why are people pretending this was not the case? Are we just going to make secborg 2.0 and remove yet another module?

**Comment by Shadowlight213** [comment]
@xxalpha probably. Then someone will make a secborg 3.0, then 4.0 and so on.
We'll end up with like 50 disabled borg modules sitting unmaintained.

**Comment by PKPenguin321** [comment]
Unless we stop merging them

**Comment by optimumtact** [comment]
No buffs to peacekeeper borgs.

Borgs will never again be effective in a fight unless emagged.

Good night
